3rdparty/musl/oe_init_c.c

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include "libc.h"
+#include "unistd.h"
+
+#include <openenclave/internal/defs.h>
+
+static size_t __auxv;
+
+void OE_CONSTRUCTOR __oe_init_c(void)
```

Comment:
Marking this as a _GCC constructor_ would ensure that this function is called via [oe_call_init_functions](https://github.com/openenclave/openenclave/blob/71edd2cf92f52017410d5e3262628b962067560a/enclave/core/init_fini.c#L58).

Do we need additional guarantee that ___oe_init_c_ must be the first global constructor that would be called.

Reply:
good point. A priority can be passed to `__attribute((constructor))__` to ensure __oe_init_c is called first. Currently I think this only matters for C++ enclaves.